backend/src/main/java/com/google/backend/servlets/AlertConfigurationServlet.java

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.DatastoreFailureException;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+
+
+/** Servlet that stores and fetches conifigurations in Datastore */
+@WebServlet("/api/v1/configurations")
+public class AlertConfigurationServlet extends HttpServlet {
+
+  private static final String EMPTY_BODY_ERROR = "No data was sent in HTTP request body.";
+  private static final String WRONG_ALERT_DATA = "Incorrect alert data sent in HTTP request.";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    List<String> results = new ArrayList<String>();
+
+    Query query = new Query("Configuration");
```

Comment:
nit: make `"Configuration"` a constant

Reply:
Could you please explain?

Reply:
you can add `"Configuration"` as a constant variable:
```
private static final String CONFIG_PROPERTY = "Configuration";
```
Since you will likely use string `"Configuration"` again somewhere else, I think it's better to put it as a constant. 

hopefully this is more clear :)